auth/src/main/java/org/aerogear/mobile/security/Check.java

```diff
@@ -0,0 +1,25 @@
+package org.aerogear.mobile.security;
+
+import org.aerogear.mobile.security.checks.RootedCheck;
+
+public enum Check {
```

Comment:
Check as in "security check" ?

Reply:
Yeah, I guess this could be `SecurityCheckType` or something like that